Assignment2

```diff
@@ -0,0 +1,56 @@
+#include<iostream>
+using namespace std;
+
+__global__ void Add(float *d_a,float *d_b,float *d_c,int r,int c){
+    
+    int i =blockIdx.x*blockDim.x+threadIdx.x;
+    int j =blockIdx.y*blockDim.y+threadIdx.y;
+    int k = i+j*c;
+    if(i<r && j<c){
```

Comment:
This if condition should be (i<c && j<r). 
This is because you have defined i as the index in the x dimension (horizontal direction) and the maximum index limit in the x dimension is the number of columns of the matrix.  And similarly the maximum limit for j (defined as the index in y dimension) is the number of rows in the matrix.

Reply:
yeah...sorry..my bad